skimage/io/tests/test_io.py

```diff
@@ -47,6 +48,7 @@ def test_imread_file_url():
     assert image.shape == (512, 512)
 
 
+@pytest.mark.skipif(is_wasm, reason="no access to pytest-localserver")
```

Comment:
I am unsure of this – could not get it to pass inside Node. Maybe it's running in a web worker already and cannot access a server? Also, `pytest-localserver` isn't available in Pyodide for some reason (even though it is a pure Python package).

Reply:
That's great. Let's skip them, then :)